src/main/java/uk/gov/justice/digital/service/DomainOperations.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.service;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.digital.domain.model.TableInfo;
+import uk.gov.justice.digital.domain.model.TableTuple;
+import uk.gov.justice.digital.exception.DomainExecutorException;
+
+public abstract class DomainOperations {
```

Comment:
Make this a singleton and have micronaut inject it via constructor injection where needed.

Reply:
removed this class since the purpose of this class is lost now